apps/frontend/src/pages/root/components/user-wheel/libs/components/retake-quiz-modal/retake-quiz-modal.tsx

```diff
@@ -2,19 +2,21 @@ import { QuizCategoriesForm } from "~/libs/components/components.js";
 import { AppRoute } from "~/libs/enums/enums.js";
 import { useAppDispatch, useCallback } from "~/libs/hooks/hooks.js";
 import { actions as appActions } from "~/modules/app/app.js";
-import {
-	type CategoriesGetRequestQueryDto,
-	actions as quizActions,
-} from "~/modules/quiz/quiz.js";
+import { actions as quizActions } from "~/modules/quiz/quiz.js";
 
 import styles from "./styles.module.css";
 
 const RetakeQuizModal: React.FC = () => {
 	const dispatch = useAppDispatch();
 
 	const handleSubmit = useCallback(
-		(payload: CategoriesGetRequestQueryDto): void => {
-			void dispatch(quizActions.getQuestionsByCategoryIds(payload));
+		(payload: number[]): void => {
```

Comment:
Don't remember doing this, should investigate if we still new CategoriesGetRequestQueryDto